syntax = "proto3";

package srv;

import "Foo.proto"; 

message FooRequest {
    string name = 1;
}

message FooResponse {
    Foo payload = 1;
}

service FooService {
  rpc GiveMeAFoo(FooRequest) returns (FooResponse);
}
